pgex/examples/simple_projectile/bullet.py

```diff
@@ -0,0 +1,32 @@
+"""
+This file is a part of the 'Pygame Examples (pgex)' source code.
+The source code is distributed under the MIT license.
+"""
+
+import pygame
+
+from ._types import PositionLike, VelocityLike
+
+
+class Bullet(pygame.sprite.Sprite):
+    image = pygame.Surface((16, 16))
+    image.fill("red")
+
+    bounding_box = pygame.Rect(0, 0, 640, 360)
+
+    def __init__(self, start_pos: PositionLike, velocity: VelocityLike) -> None:
+        """Initializes bullet."""
+
+        super().__init__()
+        self.start_pos = pygame.Vector2(start_pos)
+        self.velocity = pygame.Vector2(velocity)
+        self.rect = self.image.get_rect(center=start_pos)
+
+    def update(self) -> None:
+        """Adds bullet's velocity to position, thus moving it."""
+
+        self.rect.topleft += self.velocity
+        if not self.rect.colliderect(
+            self.bounding_box
```

Comment:
Since `bounding_box` is a class attribute (as well as `image`) I would refer to it using `Bullet.bounding_box` instead to not letting readers being confused and think that it could be a regular instance attribute.